src/ui/public/indices/get_indices.js

```diff
@@ -1,6 +1,10 @@
 import { pluck, reduce, size } from 'lodash';
 
-const getIndicesFromResponse = json => {
+const getIndexNamesFromAliasesResponse = json => {
+  if (json.status === 404) {
```

Comment:
Hmm. I'm not so sure I agree, but I don't mind it living here too if you feel strongly about it